### PR TITLE
Instructions on documenting code style modifications

### PR DESCRIPTION
**Pull request by felix-hilden**
From #2106: @ichard26 I know, *I know* you said you'd end up taking care of it yourself, but it's such an easy addition.

This PR adds a section to `CONTRIBUTING.md` where any style changes should be documented in the *Black* code style document. Probably another `skip news`?

Any wording improvements welcome, and I'll leave this open for modification in case you'd like to finish it yourself! Or if you had something else in mind, feel free to close this outright.

### CODE REVIEW

**Review by ichard26 — APPROVED** [review]
> From #2106: @ichard26 I know, I know you said you'd end up taking care of it yourself, but it's such an easy addition.

Well, there's a reason why I put "probably" in my wording :)

> another `skip news`?

mhmm you got it!

SGTM. Many thanks for picking this up and being so easy and fun to work with!

**Comment by felix-hilden** [comment]
Much appreciated 😄 but just so you know... [you started it](https://github.com/psf/black/issues/53#issuecomment-818883474) (hehe). But in all seriousness, thanks for being so open to work with us newbies!

**Comment by felix-hilden** [comment]
Just so that I'm on the same page: are we waiting for another maintainer to rewiew this? Or what's the pr-review-approve-merge etiquette of Black?

**Comment by ichard26** [comment]
> what's the pr-review-approve-merge etiquette of Black?

Honestly, no clue! I'm the newest maintainer and we're quite informal around here. Any sort of "etiquette" around here is just implicitly agreed upon and does change. There are a few things that I've noticed though that may be informative:

1. One approval by a maintainer is enough
    - Although each maintainer has a field they review best (e.g. documentation is my cup of tea), so it's better if the PR is 
      approved by them.
2. If there's any "requesting changes" reviews from a maintainer, they must be addressed before merge
3. If you want others to wait for a review from you, "self-requesting" a review may or may not be enough (explicitly saying "HEY, I wanna review this, please wait" is best)
4. We do wait before merging a PR once it's been approved to give others a chance to raise any comments

Perhaps writing down some formalities would be a good idea :ponders:

**Comment by felix-hilden** [comment]
I see, many thanks! And for the merge as well 🎉

**Pull request merged**
